src/app/shared/filter-bar/filter-bar.component.css

```diff
@@ -11,12 +13,29 @@
 
 .dropdown-filters {
   width: 100%;
+  display: flex;
+  flex-wrap: nowrap;
 }
 
 .label-filter-grid-tile {
   width: 100%;
+  justify-content: end;
 }
 
 .submit-button {
   margin-left: 8px;
 }
+
+.dropdown-filters-container {
+  margin-right: 8px;
+  overflow-x: auto;
+  display: flex;
+  justify-content: center;
+  scrollbar-width: 5px;
```

Comment:
Should this be using the firefox specific styling? 
It would be better to use webkit:scrollbars pseudo-class selector in addition to this to properly style scrollbars in all browsers.
See https://stackoverflow.com/questions/15420314/how-to-make-custom-scrollbars-show-in-all-browsers